pcawg_metadata_parser/parse_gnos_xml.py

```diff
@@ -1575,7 +1756,14 @@ def bam_aggregation(bam_files):
                         alignment_status.get('tophat').get('gnos_info').get('gnos_repo').append(bam['gnos_repo'])
                         alignment_status.get('tophat').get('gnos_info').get('gnos_last_modified').append(bam['last_modified'])
                 else:
-                    logger.warning( 'Same aliquot: {} from donor: {} using same workflow: {} has different aligned GNOS BAM entries, in use: {}, additional: {}'
+                    if bam['is_santa_cruz_entry']:
+                        aliquot_tmp = create_aggregated_rna_bam_info(bam)
+                        alignment_status['tophat'] = aliquot_tmp
```

Comment:
not sure why the above two statements can not be merged into one, why need aliquot_tmp variable at all